__tests__/supergraph/errors/EXTERNAL_MISSING_ON_BASE.spec.ts

```diff
@@ -97,6 +97,56 @@ testVersions((api, version) => {
         },
       ]),
     );
+
+    assertCompositionSuccess(
+      api.composeServices([
+        {
+          name: "a",
+          typeDefs: graphql`
+            extend schema
+              @link(
+                url: "https://specs.apollo.dev/federation/v2.3"
+                import: ["@key", "@external", "@requires"]
+              )
+
+            extend type SharedType @key(fields: "id") {
+              id: ID! @external
```

Comment:
This field is only declared here. This passes apollo composition.

However our composition library spits out:

```
EXTERNAL_MISSING_ON_BASE Field "SharedType.id" is marked @external on all the subgraphs in which it is listed (subgraph "a").
```